### PR TITLE
Feature/update from r3.5 to 4

### PR DESCRIPTION
**Pull request by Surekha-HMCTS**
### Change description ###

Latest from release 3.5 branch to 4

### CODE REVIEW

File: src/app/components/routing/application-routing.component.ts [hunk 1]
```diff
@@ -1,29 +1,25 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { select, Store } from '@ngrx/store';
-import { combineLatest, iif, Observable, Subscription } from 'rxjs';
-import { of } from 'rxjs/internal/observable/of';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { combineLatest, Observable, iif, Subscription } from 'rxjs';
```
Comment (connorpgpmcelroy) [inline src/app/components/routing/application-routing.component.ts:5]: iif?

File: src/cases/containers/case-viewer-container/case-viewer-container.component.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import * as fromRoot from '../../../app/store';
 import { WASupportedJurisdictionsService } from '../../../work-allocation/services';
 import {FeatureVariation} from '../../models/feature-variation.model';
 import {Utils} from '../../utils/utils';
+import { WAFeatureConfig } from 'src/work-allocation/models/common/service-config.model';
```
Comment (connorpgpmcelroy) [inline src/cases/containers/case-viewer-container/case-viewer-container.component.ts:16]: src

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Two minor changes to request

**Review by connorpgpmcelroy — APPROVED** [review]
Seems all good to me

**Review by DavidJayakumar — APPROVED** [review]

**Pull request merged**